test/acceptance/DatesSpec.scala

```diff
@@ -5,7 +5,7 @@ import utils.Dates
 
 class DatesSpec extends FeatureSpec {
   feature("Date Formatters") {
-    scenario("Ordinal day formatting", Acceptance) {
+    scenario("Ordinal day formatting") {
```

Comment:
Feels like this is in the wrong place then? Should it be in https://github.com/guardian/subscriptions-frontend/tree/master/test/utils